iOS/README.md

```diff
@@ -0,0 +1,10 @@
+## ONNXRuntime iOS Sample 
```

Comment:
```suggestion
# **WARNING: THIS EXAMPLE HAS NOT YET BEEN VERIFIED TO WORK WITH THE LATEST VERSION OF ONNX RUNTIME.**

## ONNXRuntime iOS Sample 
```

consider adding an obvious warning like the one above for examples which have yet to be verified and covered by CI tests.
perhaps also here:
swift/readme.md
python/readme.md (doesn't exist yet)
c_sharp/readme.md (doesn't exist yet)

Reply:
Maybe we can move all these API examples under a directory, say API_examples

Reply:
@gwang-msft , you mean all the samples I'm copying in?  I can do it. @edgchen1, what do you think?

Reply:
I don't have a strong opinion on whether it's in a subdirectory or not. Regarding the name "API_examples", the other examples show example API usage too. Is there a more specific name?

Reply:
So, what are API examples and what are not? Sorry, what are the existing samples for in this repo?

Reply:
I don't have a strong opinion either, other examples here usually have a specific target platform, such as web or mobile, the ones added in the PR are more like examples on how to use a certain API such C++, C#